src/server/callback/listener/handle-scan-results-callback.js

```diff
@@ -1,49 +1,52 @@
 import { toScanResultResponse } from '~/src/server/common/helpers/scan-result-response'
 import { deleteSqsMessage } from '~/src/server/common/helpers/sqs/delete-sqs-message'
-import {
-  isAcknowledged,
-  uploadStatus
-} from '~/src/server/common/helpers/upload-status'
 import fetch from 'node-fetch'
 
 async function handleScanResultsCallback(message, callbackQueueUrl, server) {
-  try {
-    const receiptHandle = message.ReceiptHandle
-    const payload = JSON.parse(message.Body)
-    const uploadDetails = await server.redis.findUploadWithFiles(
-      payload.uploadId
+  const receiptHandle = message.ReceiptHandle
```

Comment:
This I feel is more preference. Theres a lot of if/else/else if statements in here that make it hard to follow. A few potential ways of making this easier to follow:

- Just use `if`, don't nest conditionals, so avoid using `else` or `else if` statements
- Abstract things out to local functions so its simple to see what a body of work is doing by the name of the `function/method`